src/main/java/org/spongepowered/common/event/SpongeEventManager.java

```diff
@@ -411,16 +414,18 @@ private boolean post(Event event, List<RegisteredListener<?>> handlers) {
             try (CauseStackManager.StackFrame frame = Sponge.getCauseStackManager().pushCauseFrame();
                  final PhaseContext<?> context = PluginPhase.Listener.GENERAL_LISTENER.createPhaseContext()
                             .source(handler.getPlugin())
-                            .buildAndSwitch()) {
-                handler.getTimingsHandler().startTimingIfSync();
+                            .buildAndSwitch();
+                 final Timing timings = handler.getTimingsHandler()) {
```

Comment:
extra space?

Reply:
Same as here, note how it lines up with `CauseStackManager`

Reply:
I guess he referred to it being 5 spaces instead of 4 or 8. IIRC 8 is a common standard for this. Aligning it as it is here is also used but less common. As always: styles are just personal preferences.

Reply:
yeah, looked really odd to me